src/avgScrabbleScoreFromFile.js

```diff
@@ -11,11 +11,20 @@ let avgScrabbleScore = require('./avgScrabbleScore');
  * sample-words.txt.
  */
 function avgScrabbleScoreFromFile(fileName) {
-  // This is your job. :)
+  let fileWords = fs.readFileSync(fileName, 'utf-8');
+  let splittedFileWords = fileWords.split('\r\n');
```

Comment:
This will only work on Windows. Depending on how you write `scrabbleScore`, splitting on just `\n` will work on every OS, but `\r\n` won't work on anything but Windows.

If you want to reference the system-specific newline string you can use `os.EOL`:, which will return the correct string for whatever operating system your program is running on: https://nodejs.org/api/os.html#os_os_eol

There's also a small bug here. Try finding the average score of a simple file like:

```text
a
a
a
```